intake-catalogs/climate.yaml

```diff
@@ -0,0 +1,20 @@
+plugins:
+  source:
+    - module: intake_xarray
+    - module: intake_esm
+
+sources:
+
+  cmip6_gcs:
+    args:
+      esmcol_path: "https://raw.githubusercontent.com/NCAR/intake-esm-datastore/master/catalogs/pangeo-cmip6.json"
+    description: 'CMIP6 in Google Cloud Storage'
+    driver: intake_esm.esm_datastore
+    metadata: {}
+
+  cmip6_s3:
```

Comment:
Isn't this a typo? `cmip6_s3` ---> `cesm1le_s3`??

Reply:
yes!  whoops should have waited to merge

Reply:
Is `cesm1le` really the preferred identifier for this dataset?

Reply:
> Is `cesm1le` really the preferred identifier for this dataset?

Not really :) It was just a suggestion. I heard that there are plans for CESM(v2) Large Ensemble. I figured it might be helpful to at least specify which version of CESM, the large ensemble was produced with.

Reply:
Why don't you make a PR with NCAR's preferred description (v1 vs v2) and standard identifier. That would be great!

Reply:
Will do!